org.eclipse.scout.rt.mom.jms/src/main/java/org/eclipse/scout/rt/mom/jms/JmsMomImplementor.java

```diff
@@ -128,6 +127,11 @@ public class JmsMomImplementor implements IMomImplementor {
    */
   public static final String JMS_REQUEST_CANCELLATION_MESSAGE_CONSUMER_JOB_RECEIVE_TIMEOUT = "scout.mom.jms.requestCancellationMessageConsumerJobReceiveTimeout";
 
+  /**
+   * Key to set {@link #m_subscriptionAwaitStartedSeconds}
+   */
```

Comment:
Maybe add a comment indicating that the absence of a configured value or a value of -1 or null means "infinite"